txp/Pos/Txp/Toil/Utxo/Functions.hs

```diff
@@ -205,6 +206,8 @@ verifyKnownInputs VTxContext {..} resolvedInputs TxAux {..} = do
                 redVer = scrVersion twRedeemer
             when (valVer /= redVer) $
                 throwError $ WitnessScriptVerMismatch valVer redVer
+            when (valVer /= vtcAdoptedVersion) $
```

Comment:
It should be `valVer > vtcAdoptedVersion`. It's ok to use script version less than the adopted one.